specs/webhook/webhook_helm-api-spec.yaml

```diff
@@ -0,0 +1,146 @@
+openapi: "3.0.3"
+info:
+  version: 1.0.0
+  title: Devtron Labs
+paths:
+  /orchestrator/webhook/helm/app:
+    post:
+      description: Create or Update helm application
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/HelmAppCreateUpdateRequest"
+      responses:
+        "200":
+          description: Create or Update helm application response
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HelmAppCreateUpdateResponse"
+        "401":
+          description: If the user is not authenicated, then this error is thrown
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HelmAppCreateUpdateResponse"
+        "403":
+          description: If the user is not authorized, then this error is thrown
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HelmAppCreateUpdateResponse"
+        "400":
+          description: If request is not correct, then this error is thrown
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HelmAppCreateUpdateResponse"
+        "500":
+          description: This error is thrown in case of some internal server error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/HelmAppCreateUpdateResponse"
+components:
+  schemas:
+    HelmAppCreateUpdateRequest:
+      type: object
+      properties:
+        clusterName:
+          type: string
+          description: Name of the cluster configured on devtron UI, if it is not provided - will be assumed as default_cluster
```

Comment:
`clusterName` should be mandatory for user

Reply:
done